File: 1045.c
#include <stdio.h>

int main(){

    float a, b, c, d;

    scanf("%f %f %f", &a, &b, &c);

    if(b > a && b > c){

        d = a;
        a = b;
        b = d;
    }else
    if(c > a){

        d = a;
        a = c;
        c = d;
    }

    if(a >= (b+c)){

        printf("NAO FORMA TRIANGULO\n");
    }else{

        if((a * a) == (b * b) + (c * c)) printf("TRIANGULO RETANGULO\n");
        if((a * a) > (b * b) + (c * c)) printf("TRIANGULO OBTUSANGULO\n");
        if((a * a) < (b * b) + (c * c)) printf("TRIANGULO ACUTANGULO\n");
        if(a == b && a == c) printf("TRIANGULO EQUILATERO\n");
        if((a == b && a != c) || (a == c && a != b) || (b == c && b != a)) printf("TRIANGULO ISOSCELES\n");

    }

    return 0;

}

File: Iniciante/1045.java
import java.util.Scanner;

public class Main {

    public static void main(String[] args) {

        Scanner scanf = new Scanner(System.in);

        float a = scanf.nextFloat();
        float b = scanf.nextFloat();
        float c = scanf.nextFloat();
        float d;

        if(b > a && b > c){

            d = a;
            a = b;
            b = d;
        }else
        if(c > a){
    
            d = a;
            a = c;
            c = d;
        }
    
        if(a >= (b+c)){
    
            System.out.println("NAO FORMA TRIANGULO");
        }else{
    
            if((a * a) == (b * b) + (c * c)) System.out.println("TRIANGULO RETANGULO");
            if((a * a) > (b * b) + (c * c)) System.out.println("TRIANGULO OBTUSANGULO");
            if((a * a) < (b * b) + (c * c)) System.out.println("TRIANGULO ACUTANGULO");
            if(a == b && a == c) System.out.println("TRIANGULO EQUILATERO");
            if((a == b && a != c) || (a == c && a != b) || (b == c && b != a)) System.out.println("TRIANGULO ISOSCELES");    
        }
    }
}